HA/xcatha.py

```diff
@@ -18,13 +22,14 @@
 #               -v      virtual ip address
 #               -n      virtual ip hostname
 #               -m      netmask for the virtual ip address,
-#                       default to 255.255.255.0
-#               -t      target database type, it can be postgresql, default is sqlite
+#                       default is 255.255.255.0
+#               -t      target database type, it can be postgresql or sqlite, default is sqlite
```

Comment:
I would suggest that we use `docopt` instead of writing our own parse. 

If users want to use this script "as is" they would just resolve the python depenedency with pip.  @robin2008 thoughts?